sdk/lib/drivers/sound/mmebuddy/mmewrap.c

```diff
@@ -365,3 +365,85 @@ MmeGetPosition(
     return Result;
 }
 
+
+MMRESULT
+MmeGetVolume(
+    _In_ MMDEVICE_TYPE DeviceType,
+    _In_ DWORD DeviceId,
+    _In_ DWORD_PTR PrivateHandle,
+    _Out_ PDWORD_PTR pdwVolume)
+{
+    MMRESULT Result;
+    PSOUND_DEVICE_INSTANCE SoundDeviceInstance;
+    PSOUND_DEVICE SoundDevice;
+    PMMFUNCTION_TABLE FunctionTable;
+
+    /* Sanity check */
+    SND_ASSERT(DeviceType == AUX_DEVICE_TYPE ||
+               DeviceType == MIDI_OUT_DEVICE_TYPE ||
+               DeviceType == WAVE_OUT_DEVICE_TYPE);
```

Comment:
Just a question out of curiosity: Was this ever hit during any of your real-world-tests?

Reply:
No, it was never hit for me at all, since it should never be hit, because it is not used for other device types. I kept this assert just for sure.

Reply:
Thank you for the answer! Resolved.